general_superstaq/check/configs.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import argparse
+import difflib
+import os
+import sys
+import textwrap
+from typing import List, Tuple
+
+from general_superstaq.check import check_utils
+
+
+@check_utils.enable_exit_on_failure
+def run(
+    *args: str,
+    config_file: str = "setup.cfg",
+    ignore_match: str = "# REPO-SPECIFIC CONFIG",
+    silent: bool = False,
+) -> int:
+
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.description = textwrap.dedent(
+        f"""
+        Checks that the check script configuration file ({config_file}) is consistent across repos.
+
+        Exceptions in {config_file} can be flagged by adding "{ignore_match}" to the line.
+        """
+    )
+    parser.parse_args(args)  # placeholder parsing to enable printing help text
+
+    # identify the "original" config file, and the file that is supposed to be a copy
+    file_orig = os.path.join(check_utils.get_root_dir(__file__), config_file)
+    file_copy = os.path.join(check_utils.root_dir, config_file)
+    lines_orig = open(file_orig, "r").read().splitlines()
+    lines_copy = open(file_copy, "r").read().splitlines()
+
+    # collect differences between config files, ignoring lines in file_copy that match ignore_match
+    matcher = difflib.SequenceMatcher(a=lines_orig, b=lines_copy)
+    deltas: List[Tuple[str, int, int, int, int]] = []
+    for tag, orig_start, orig_end, copy_start, copy_end in matcher.get_opcodes():
+        if tag == "equal":
+            continue
+        diff_lines_copy = lines_copy[copy_start:copy_end]
+        if not diff_lines_copy or any(ignore_match not in line for line in diff_lines_copy):
+            deltas.append((tag, orig_start, orig_end, copy_start, copy_end))
+
+    if not deltas:
+        if not silent:
+            print(check_utils.success("Config files are consistent!"))
+        return 0
+
+    # print diffs
+    print(check_utils.failure(f"Found {len(deltas)} differences between config files:"))
+    print(f"{file_orig} (original)")
+    print(f"{file_copy} (copy)")
```

Comment:
nit
```suggestion
    print(check_utils.styled(f"< {file_orig} (original)", check_utils.Style.RED))
    print(check_utils.styled(f"> {file_copy} (copy)", check_utils.Style.GREEN))
```
to match the text below (similar to the output of `diff`)